.github/workflows/linux.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           --icon app/assets/linux/icon-linux.png \
           --java-options '"-Djdk.gtk.version=2"' \
           --java-options '"--add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED"' \
+          --java-options '"-Djava.library.path=/opt/scenebuilder/lib/runtime/bin:/opt/scenebuilder/lib/runtime/lib"' \
```

Comment:
why not just use:

```
--java-options "-Djava.library.path=runtime/bin;runtime/lib"
```

This way it can move into `jpackage.sh` and used for both Linux and Mac

Reply:
I have tried this and actually it did not work. As the installation location of SB seems to be fixed for this kind of package, I decided to go with hard coded paths. The generally more preferable approach is actually the one you suggested.

Reply:
I would consider a generalization here as a separate PR. What do you think?

Reply:
Not yet. Well, this approach still prevents from loading the wrong libraries. But, this was also the reason to have the "About" dialog to show the effective library paths and library paths which do not exist. Don't know yet, how to get it right with relative paths on Linux.

Reply:
Since the install dir is also declared in the same command, I guess this approach is OK for now.